test/Recipe-test.js

```diff
@@ -0,0 +1,56 @@
+import { expect } from 'chai';
+import Recipe from '../src/classes/Recipe';
+import { sampleIngredientsData, sampleUsersData, sampleRecipeData } from '../src/data/sample-data';
+
+describe('Recipe', () => {
+  sampleIngredientsData
+  sampleUsersData
+  sampleRecipeData
+  let recipe
+
+
+  beforeEach(() => {
+    recipe = new Recipe(sampleRecipeData[0]);
```

Comment:
You may pass a whole object in the argument of Recipe so we can see the data. Example: recipe = new Recipe({id: 20081, name: wheat flour, estimatedCostInCents: 142})